lib/matplotlib/image.py

```diff
@@ -890,7 +819,7 @@ def __init__(self, ax,
         self.update(kwargs)
         self.set_data(x, y, A)
```

Comment:
Shouldn't this constructor be doing `super(PcolorImage, self).__init__(...)`? Right now, it is skipping the `AxesImage` constructor even though that is its parent.